online-inference/stable-diffusion/02-model-download-job.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: stable-diffusion-download
+spec:
+  template:
+    spec:
+      containers:
+      - name: model-downloader
+        image: tweldoncw/model-downloader:10
+        imagePullPolicy: IfNotPresent
+        command: 
+          - "python3"
+          - "/app/download.py"
```

Comment:
Did you implement your own model downloader instead of extending the one we had? :)

Reply:
Yes, in the interest of time since there is no `huggingface_hub` lib for Go, and `diffusers` requires the `huggingface_hub` directory structure as described [here](https://huggingface.co/docs/huggingface_hub/v0.9.0.rc0/en/package_reference/file_download#caching). Also I'm more familiar with Python than I am Go :)

Reply:
@Oogy No problem, but we'll need to fix that. ;)